### DISCUSSION TITLE
User name and description save?

### DISCUSSION THREAD

**Discussion by mykeehu** [oobabooga/text-generation-webui#6274] [Q&A] [answered]
Where and how can the user name and user description be saved from the UI? I found a name1 field in the settings.yaml file that I could manually overwrite, but I couldn't find a field for the user description. Or how to save them in the interface, because there is no button.

**Comment by Gamefriend996** [comment]
Same here.

**Comment by weirdnumbat** [comment, marked as answer]
After editing User's name & description, go to 'Session' -> 'Save UI defaults'
